### DISCUSSION TITLE
A problem about dropout_adj

### DISCUSSION THREAD

**Discussion by fxsxjtu** [pyg-team/pytorch_geometric#3717] [Q&A]
When I try to use the dropout_adj , I encoutered a basic problem.
`adj, features, labels, idx_train, idx_val, idx_test = load_benchmark("cora")
a,b = dense_to_sparse(adj)
a,b = dropout_adj(edge_index=a,p=0.95,edge_attr=b)
ab = to_dense_adj(edge_index=a,edge_attr=b)`
I just want to dropout some edges in adj , but the size of the adj is also changed!
If there is any way to keep the size while dropping the edges?I'd appreciate some help.

**Comment by wsad1** [comment]
What might be happening is that `dropout_adj` is dropping edges between the last few nodes, and when you call `to_dense_adj` its not able to infer the number of nodes in the graph correctly. Try setting the `batch`  or `max_num_nodes` parameter in `to_dense_adj`.

**Reply by fxsxjtu** [reply]
Thanks alot ,it does work!
